app/react/Documents/components/DocumentSidePanel.js

```diff
@@ -368,11 +371,11 @@ export class DocumentSidePanel extends Component {
               <TabContent for="toc" className="toc">
                 <div className="tocHeader">
                   <h1>
-                    <Translate>Table of contents </Translate>
+                    <Translate>Table of contents</Translate>
                   </h1>
                   &nbsp;
                   <TocGeneratedLabel file={this.props.file}>
-                    <Translate>auto-created ⓘ </Translate>
+                    <Translate>auto-created ⓘ</Translate>
```

Comment:
Do we want to translate with the small icon?

Reply:
moved the icon outside